sycl/doc/extensions/DeviceInfoWorkSizes/README.md

```diff
@@ -0,0 +1,56 @@
+# Device information descriptor: ext_oneapi_max_number_work_groups
```

Comment:
I'd suggest a few changes to the format of this extension specification document.  In general, use the [device-if](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/DeviceIf/device_if.asciidoc) extension as a template, but here are some specific comments:

* Use the name of the feature-test macro as the title of the document.
* Name the file itself according to the name of the feature-test macro, for example if the feature-test macro name is `SYCL_EXT_ONEAPI_WORK_GROUP_QUERY`, the name of the file would be "work_group_query.md"

The general outline for the spec should look like:

* Introduction (provide a motivation for why this feature is needed)
* Section describing the feature-test macro
* Section describing the new `info::device` descriptor(s).  Maybe add a table here with three columns: descriptor name, return type, description.
* Section showing an example usage (which you have already)
* If warranted, a section describing implementation issues.

I see that the implementation adds two info descriptors, but only one is described in the specification document.  Please update the spec to describe them both.